root/collection/CollectionHeader.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l} from '../static/scripts/common/i18n';
+import EntityTabLink from '../components/EntityTabLink';
+import SubHeader from '../components/SubHeader';
+import Tabs from '../components/Tabs';
+import EditorLink from '../static/scripts/common/components/EditorLink';
+import EntityLink from '../static/scripts/common/components/EntityLink';
+import {withCatalystContext} from '../context';
+
+
+type Props = {|
+  +$c: CatalystContextT,
+  +collection: CollectionT,
+  +page: string,
+|};
+
+const CollectionHeader = ({$c, collection, page}: Props) => {
+  const subHeading = (
+    <>
+      {collection.public ? (
+        l('Public collection by {owner}',
+          {owner: <EditorLink editor={collection.editor} />})
+      ) : l('Private collection')}
+      {collection.editor ? (
+        <span className="small">
+          {[
```

Comment:
I was gonna ask why not use a fragment, but actually, why not use `bracketed` instead. :P

Reply:
Changed. I've been wondering - we have many cases of having to add {' '} before a parens or whatever - should we have a spaceBefore() function instead?

Reply:
Dunno, `{spaceBefore(...)}` doesn't seem like a real improvement over `{' '}{...}`, but that's just me. :)